### DISCUSSION TITLE
Suspend support

### DISCUSSION THREAD

**Discussion by MattBystrin** [ArthurSonzogni/FTXUI#309] [Ideas]
Hi, @ArthurSonzogni ! 
Today I've noticed that ScreenInteracitve do not works with SIGSTP. I've rarely use ctrl+z in term in case i usualy work with tmux, but maybe someone finds it handy. What do you think ?

**Comment by ArthurSonzogni** [comment]
Yes, I believe we should support it.

**Reply by ArthurSonzogni** [reply]
This is part of v3.0.0
